pkg/image/registry/image/etcd/etcd.go

```diff
@@ -59,8 +59,27 @@ func NewREST(optsGetter restoptions.Getter) (*REST, error) {
 	}
 
 	if err := restoptions.ApplyOptions(optsGetter, store, prefix); err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
-	return &REST{store}, nil
+	internalStore := *store
+	internalStore.UpdateStrategy = image.InternalStrategy
+	internalStore.CreateStrategy = nil
+
+	return &REST{store}, &InternalREST{store: &internalStore}, nil
+}
+
+// InternalREST implements a RESTStorage for image resources internally modifiable.
```

Comment:
> As discussed before, we don't want to allow to set signatures during image
> push nor image import.

We curate those endpoints.  I'm fine with the curated endpoints stripping fields to limit their scope.

Since we're going to loopback to the API server to go through proper admission/defaulting flows (https://github.com/openshift/origin/pull/9181#discussion_r70851489), the images REST endpoint will have the power to make these changes.  The interface there can't perform that restriction, but the REST endpoint calling can do so carefully.  I think is implementation already restricts it sufficiently.